packages/mui-material/src/Menu/Menu.js

```diff
@@ -156,6 +159,23 @@ const Menu = React.forwardRef(function Menu(inProps, ref) {
     }
   });
 
+  const PaperSlot = slots.paper ?? MenuPaper;
+  const paperExternalSlotProps = slotProps.paper ?? PaperProps;
+
+  const rootSlotProps = useSlotProps({
+    elementType: slots.root,
+    externalSlotProps: slotProps.root,
+    ownerState,
+    className: clsx(classes.paper, className),
```

Comment:
It should be `classes.root` for the root

```suggestion
    className: [classes.root, className],
```

This should solve the failing CI tests